src/controller/citations-controller.ts

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios';
+import { NextFunction, Request, Response } from 'express';
+
+export const citationsController = () => {
+  const downloadBibtex = async (req: Request, res: Response, next: NextFunction) => {
+    const {
+      publisherId,
+      articleId,
+    } = req.params;
+    const doi = `${publisherId}/${articleId}`;
```

Comment:
Do, we want to restrict requests to this endpoint to doi's that are recognised in server? Should we return a 404 if the doi is not found on our server rather than as a result of the request to crossref?